src/merkle/error.rs

```diff
@@ -0,0 +1,55 @@
+use core::fmt;
+
```

Comment:
Optionally remove this line.  I think we should try and conform on standards around this as I often like to keep empty lines similar to how [isort](https://pycqa.github.io/isort/) arranges imports.  However, I think the current standard is to not have empty lines.

Reply:
I think I've come around on this. I'm fine with using one of the standard options - we just need to decide which one.